src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaDetail/mediaDetail.scss

```diff
@@ -2,4 +2,26 @@
 
 .media-detail {
     padding: $viewPadding;
+    display: flex;
+
+    @media (max-width: 720px) {
```

Comment:
Have you decided the breakpoint on your own? `720px` seem pretty small: 
![image](https://user-images.githubusercontent.com/405874/33956517-13bb56b8-e03f-11e7-9095-8e97a2435b91.png)